pkgs/by-name/sc/scrounge-ntfs/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, stdenv
+, fetchurl
+, autoreconfHook
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "scrounge-ntfs";
+  version = "0.9";
+
+  src = fetchurl {
+    url = "http://thewalter.net/stef/software/scrounge/scrounge-ntfs-${finalAttrs.version}.tar.gz";
+    hash = "sha256-HYrMIMTRPmgAac/vaZ1jaUFchyAl5B0quxgHH0DHJ84=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  meta = with lib; {
+    description = "Data recovery program for NTFS file systems";
+    mainProgram = "scrounge-ntfs";
+    homepage = "http://thewalter.net/stef/software/scrounge/";
+    maintainers = with maintainers; [ d3vil0p3r ];
+    platforms = platforms.unix;
+    license = licenses.unfree;
```

Comment:
home page states:
```
Open Source BSD type license
```
http://thewalter.net/git/cgit.cgi/scrounge-ntfs/tree/COPYING
so the license seems to be free, though offhand i am not sure what the proper mechanism is to specify one-of licenses.
https://nixos.org/manual/nixpkgs/unstable/#sec-meta-license

Reply:
Mmh actually also the [BlackArch PKGBUILD](https://github.com/BlackArch/blackarch/blob/master/packages/scrounge-ntfs/PKGBUILD) sets the license as BSD.

Reply:
@a-n-n-a-l-e-e used `bsd0`.

Reply:
`bsd3` https://opensource.org/license/bsd-3-clause/ matches better

Reply:
Fixed